src/lib/syncable_ledger/syncable_ledger.ml

```diff
@@ -221,6 +235,28 @@ end = struct
 
   type query = Addr.t Query.t
 
+  (* TODO: parameterize *)
+  let subtree_depth : index = 4
```

Comment:
Agree with the comment, this should be parameterised.

Reply:
About that, the initial idea was for the request to specify this depth, but as I ended up changing only the response that wouldn't be possible.
It could be set on the responder node but that would potentially allow the responder to send very big responses to make you waste time.
What did you have in mind?